src/scripts/main.js

```diff
@@ -1,3 +1,26 @@
 'use strict';
 
-// write code here
+// Беремо всі хедери у масив, щоб додати до кожного ивент
+const headers = [...document.querySelectorAll('th')];
+
+for (let i = 0; i < 4; i++) {
```

Comment:
The loop is hardcoded to iterate 4 times, which assumes there are exactly 4 headers. Consider using `headers.length` instead to make the code more flexible and adaptable to tables with different numbers of columns.